src/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Fragment } from 'react'
+
+import { Divider } from '../divider'
+import { TextCaption } from '../typography/caption'
+import { StyledBreadcrumb } from './Breadcrumb.style'
+
+// make sure it validates https://search.google.com/test/rich-results
+export type CrumbProps = Readonly<{
+  name: string
+  url: string
+}>
+
+export type BreadcrumbProps = {
+  crumbs: Array<CrumbProps>
+}
+
+export const Breadcrumb = ({ crumbs }: BreadcrumbProps) => (
+  <Fragment>
+    <StyledBreadcrumb itemScope itemType="https://schema.org/BreadcrumbList">
+      {crumbs.map((crumb, index) => {
+        const position = Number(index + 1)
+
+        return (
+          <li
+            key={`crumb-${crumb.name}-${position}`}
```

Comment:
Why do you add the `index`, isn't the name unique?

Reply:
`name` isn't enforced as unique, it sounds stupid that 2 links in breadcrumb would be called the same but I prefer to use the `position` for the uniqueness. And I prefer to keep `name` just to make it more readable.